arch/xtensa/src/esp32/Kconfig

```diff
@@ -2647,4 +2647,8 @@ config ESP32_TICKLESS
 	select ARCH_HAVE_TICKLESS
 	select SCHED_TICKLESS
 
+config ESP32_IGNORE_CHIP_REVISION_CHECK
+	bool "Ignore chip revision"
+	default n
+
```

Comment:
```suggestion
config ESP32_IGNORE_CHIP_REVISION_CHECK
	bool "Ignore chip revision check"
	default n
	depends on EXPERIMENTAL
	---help---
		Ignore chip revision check for ESP32. This is an experimental because 
		chip revisions before v3.0 are not supported in NuttX (please check
	
		https://www.espressif.com/sites/default/files/documentation/esp32_chip_revision_v3_0_user_guide_en.pdf).
```